examples/02_Parameter Estimation.ipynb

```diff
@@ -13,38 +13,54 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Parameter estimation is used to tune the parameters of a general model so its behavior matches the behavior of a specific system. Parameter Estimation can be considered more of an art than a science. For example, parameters of the battery model can be tuned to configure the model to describe the behavior of a specific battery.\n",
+    "Parameter estimation is used to tune the parameters of a general model so that its behavior matches that of a specific system. Parameter Estimation can be considered more of an art than a science. For example, the parameters of a battery model can be tuned to configure the model to more accurately describe the behavior of a specific battery.\n",
     "\n",
-    "Generally, parameter estimation is done by tuning the parameters of the model so that simulation (see 1. Simulation) best matches the behavior observed in some available data. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationall expensive. Fortunately, in this case we have a relatively simple model.\n",
+    "Generally, parameter estimation is done by tuning the parameters of the model so that the simulation (see __[01 Simulation](01_Simulation.ipynb)__) best matches the behavior observed in some available data. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationally expensive. Fortunately, in this case we have a relatively simple model.\n",
```

Comment:
```suggestion
    "Generally, parameter estimation is done by tuning the parameters of the model so that the simulation (see __[01 Simulation](01_Simulation.ipynb)__) best matches the behavior observed in some available data. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationally expensive.\n",
```
We introduce the model later